helpers.js

```diff
@@ -0,0 +1,25 @@
+import { userData } from './data/index.js';
```

Comment:
It is OK if you accidentally remove the file during rename. Maybe move the helpers.js and middleware.js into the public folder

Reply:
Otherwise LGTM